api/contrib/envoy/extensions/filters/http/checksum/v3alpha/checksum.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.checksum.v3alpha;
```

Comment:
Is this necessary to use the `v3alpha`? I think `v3` + WIP flag should be enough. Then we needn't change the `v3alpha` to `v3` in the future.

Reply:
That's a discussion to take up with the documentation, I originally used v3 then found [this directive](https://sourcegraph.com/github.com/envoyproxy/envoy/-/blob/EXTENSION_POLICY.md?L172) suggesting I should make it v3alpha.

Reply:
Although considering the fact that our API is stopped in the V3 forever, the version of API is accutally meaningless. So,
 I doubt the meaning to use the v3alpha.

But Ok.